scripts/linters/js_ts_linter.py

```diff
@@ -284,6 +291,83 @@ def _check_http_requests(self):
 
         return summary_messages
 
+    def _check_ts_ignore(self):
```

Comment:
If this also checks ts-expect-error ... do we need to update the name/docstring/printed-stuff/varnames/etc.?

Reply:
now there is a separate check for ts-expect-error.